Examples/Geometry/VR/index.js

```diff
@@ -36,8 +36,7 @@ const renderWindow = fullScreenRenderer.getRenderWindow();
 // this
 // ----------------------------------------------------------------------------
 
-const coneSource = vtkConeSource.newInstance({ height: 100.0, radius: 50.0 });
-// const coneSource = vtkConeSource.newInstance({ height: 1.0, radius: 0.5 });
+const coneSource = vtkConeSource.newInstance({ height: 1.0, radius: 0.5 });
```

Comment:
It is good to at least test using a cone with a crazy large/small size and positioned far from the origin to make sure everything works properly. I think your changes still use the same basic path as the old code so it should be fine. But testing unusual sizes and locations help find a lot of issues.

Reply:
That makes sense. The goal of changing to a smaller height/radius is so that we can test translation is working, much easier to see that we're moving around a small cone in the foreground than a very large cone in the distance.

Reply:
The reset camera method should (if all is working) look at the visible geometry and setup the physical scale and translation so that the geometry is near the viewer and at a reasonable scale. e.g. if you place a cone 1km away, ResetCamera should detect that and adjust the camera properties (physical translation) so that the viewer is near the cone. It should look like it is right next to you. Same idea with scale. Hope that makes sense.

Reply:
`resetCamera` positions the camera correctly prior to entering the VR session, but once we start the session we set the camera to the position of each eye (two passes) in each frame in the local reference space we get from WebXR representing the physical confines of the viewer's VR setup. 

For 3DOF (rotation only) we only have one vantage point so the cone looks the same when it's 1km away and large vs 1m away and small. To demonstrate 6DOF it is best to be able to walk around the cone, which requires that the cone is close to the viewer within the local reference space.

We can experiment with moving the local reference space and introducing controller interactions with the WebXR gamepad API in future updates.

Reply:
We have to modify the matrices from WebXR to do it correctly. 

With the old example the cone could be 1km away and super small and it would still work and look reasonable to the viewer. It isn't that the Cone is large and far away so it looks the same. It is that we take the matrices from the VR system and modify them by a world to physical matrix prior to using them. The world to physical matrix (stored in vtkjs as a few members on Camera) consists of a rotation, translation and uniform scale. It allows any size cone, placed in any location, to look reasonable when viewed in VR by default. And you can walk around it. 

If the VR example isn't showing up properly with the old Cone values, where you can walk around the cone, then we have something we need to fix.

To see what I mean try running the current example https://kitware.github.io/vtk-js/examples/VR/index.html and see how even through the cone is not at the origin you can walk around it.  If you change the values it still works.  I just set it to 2000 meters away with a height of 0.01 and it still looks good and I can walk around it. Because resetCamera is adjusting the physicaltoworld based on the currently loaded data. So if the Cone in the VR example only looks reasonable with some settings then we probably need to fix something.

Reply:
The reason we care about this is that most datasets are not located at the origin with a size of about 1.0. They tend to be all over the place and with sizes that can range all over the place. So we built in a mechanism where the VR code through resetCamera will come up with a reasonable view for any of these datasets by default.

Reply:
Thanks for the thorough explanation @martinken. I see what you mean after looking back at the WebVR example and it certainly seems reasonable to center arbitrary datasets for the VR view. It also seems possible that a user would want to fix the scale of a dataset for a walkthrough, but that can be revisited later if we want.

I've pushed a fixup commit that should resolve the issue, was previously calling `computeViewParametersFromViewMatrix` which bypasses the world-to-physical transformation as opposed to `computeViewParametersFromPhysicalMatrix` which correctly applies the transformation to the view matrix we receive from WebXR for each eye. The "large" cone seems reasonably scaled when testing in a headset and simulator. Please feel free to check on your side as well.

Reply:
> For 3DOF (rotation only) we only have one vantage point so the cone looks the same ("right next to you") when it's 1km away and large vs 1m away and small.

I am not sure this is true. Wouldn't the convergence of the eyes matter? The user should be able to sense the difference between these two scenes.

Reply:
Yes exactly @tbirdso  Perfect. On the C++ side we do provide VR GUI elements that allow the user to select an exact scale or to arbitrarily scale the world using the controllers. But agree that is all future MR stuff .